src/app/app.module.ts

```diff
@@ -22,6 +22,7 @@ import { UserDetailComponent } from './users/user-detail/user-detail.component';
 import { UserDetailResolver } from './_resolvers/user-detail.resolver';
 import { UserListResolver } from './_resolvers/user-list.resolver';
 import { UserRegisterComponent } from './users/user-register/user-register.component';
+import { TicketComponent } from './ticket/ticket.component';
```

Comment:
@mkorzecki Tak właściwie to powinien być folder tickets, w którym siedziałby ten komponent(jak w przypadku users) - będziemy mieli kilka komponentów do ticketów. Poza tym sama nazwa ticket niewiele mówi.